tests/integration/utils.py

```diff
@@ -125,6 +125,34 @@ def check_mismatched_images(
 
 # Inspired by https://github.com/E3SM-Project/e3sm_diags/blob/master/docs/source/quickguides/generate_quick_guides.py
 
+def get_andes_expansions(config):
+    # Note: `os.environ.get("USER")` also works. Here we're already using mache but not os, so using mache.
+    username = config.get("web_portal", "username")
+    web_base_path = config.get("web_portal", "base_path")
+    d = {
+        "bundles_walltime": "02:00:00",
+        "constraint": "",
+        # To run this test, replace conda environment with your e3sm_diags dev environment
+        # TODO: Get conda working on Andes
```

Comment:
I think you just need to log in with a new terminal after you install.  The instruction:
```
Do you wish the installer to initialize Miniconda3 by running conda init? [yes|no] yes
```
is adding something to your `.bashrc` and that doesn't take effect unless you either do `source .bashrc` or you log in to a fresh terminal.

Reply:
If that doesn't work, you should instead do:
```
source ~/miniconda3/etc/profile.d/conda.sh
source ~/miniconda3/etc/profile.d/mamba.sh
mamba activate
```
if you installed Miniconda3 instead of Mambaforge -- ☹️ -- you will need to just do:
```
source ~/miniconda3/etc/profile.d/conda.sh
conda activate
```

Reply:
> ```
> source ~/mambaforge/etc/profile.d/conda.sh
> source ~/mambaforge/etc/profile.d/mamba.sh
> mamba activate
> ```

Ah, that appears to work, thanks!